ApiaryReportingApi.md

```diff
@@ -270,9 +270,11 @@ HOST: http://api.apiary.io/
   - duration (number) ... Test duration in seconds
   - result (string) ... Result of step execution
     - Values
-      - `passed`
-      - `failed`
```

Comment:
This is only a change making the documentation conforming with reality. Code is not changed in this way.